info.json

```diff
@@ -0,0 +1,223 @@
+            }
+        ],
+        "modules": {
+            "announcements": {
+                "name": "Announcements",
+                "apiName": "announcements"
+            },
+            "devices": {
+                "name": "Devices",
+                "apiName": "devices"
+            },
+            "managers": {
+                "name": "Managers",
+                "apiName": "managers"
+            },
+            "metafield_templates": {
+                "name": "Metafield Templates",
+                "apiName": "metafield_templates"
+            },
+            "scripts": {
+                "name": "Scripts",
+                "apiName": "scripts"
+            },
+            "z_t_p_profiles": {
+                "name": "ZTP Profiles",
+                "apiName": "z_t_p_profiles"
+            }
+        },
+        "viewTemplates": {
+            "managers": {
+                "name": "Managers",
+                "apiName": "managers",
+                "views": [
+                    "list",
+                    "detail",
+                    "form"
+                ]
+            },
+            "metafield_templates": {
+                "name": "Metafield Templates",
+                "apiName": "metafield_templates",
+                "views": [
+                    "list",
+                    "detail",
+                    "form"
+                ]
+            },
+            "scripts": {
+                "name": "Scripts",
+                "apiName": "scripts",
+                "views": [
+                    "list",
+                    "detail",
+                    "form"
+                ]
+            },
+            "z_t_p_profiles": {
+                "name": "ZTP Profiles",
+                "apiName": "z_t_p_profiles",
+                "views": [
+                    "list",
+                    "detail",
+                    "form"
+                ]
+            }
+        },
+        "views": {
+            "navigation": [
+                "FortiManager"
+            ]
+        },
+        "roles": [
+            {
+                "name": "FortiManager-Playbook"
+            },
+            {
+                "name": "FortiManager-Admin"
+            }
+        ],
+        "playbooks": [
+            {
+                "name": "FortiManager(FMG)-Buttons"
+            },
+            {
+                "name": "FortiManager(FMG)-CustomPlaybooks"
+            },
+            {
+                "name": "FortiManager(FMG)-Microservices"
+            },
+            {
+                "name": "FortiManager(FMG)-PlayGround"
+            },
+            {
+                "name": "FortiManager(FMG)-RecordHandlers"
+            },
+            {
+                "name": "FortiManager(FMG)-Synchronizing"
+            },
+            {
+                "name": "FortiManager(FMG)-Triggers"
+            }
+        ],
+        "globalVariables": [
+            {
+                "id": 7,
+                "name": "Server_fqhn",
```

Comment:
@fortijames Please remove the global variable 'Server_fqdn' as its value may vary across different FSR instances.

Reply:
> @fortijames Please remove the global variable 'Server_fqdn' as its value may vary across different FSR instances.

I don't know how this is being exported. I have Global Vars unchecked.